### DISCUSSION TITLE
Log location

### DISCUSSION THREAD

**Discussion by dodds-cc** [Igalia/cog#528] [Q&A]
Hi everyone,

We're using cog in an embedded device in development and enjoying it's features so far

Quick question though, where can I find the logging output? After leaving the device running for a few days, the screen has gone black and is showing a flashing cursor. The cog process is still running but is maxxed out at 100% CPU, so I'd like to find out where it all started to go wrong to debug the JS. I see no config options to set the log location

Cheers

**Comment by philn** [comment]
It depends which kind of logs you need. Some of them are documented in this wiki page which is for the GTK port but also applies to WPE. https://trac.webkit.org/wiki/WebKitGTK/Debugging#Loggingsupport
